### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by maximeconnolly**
There was a problem with the run order of the web container. It would failed if the db container was not running and accepting connection.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,11 +5,10 @@ services:
...
   - docker-compose exec web sh -c "apk add --no-cache git"
-  - docker-compose exec web sh -c "pipenv install -d tox"
+  - docker-compose exec web sh -c "pipenv install tox"
+
 
 script:
   - docker-compose exec web sh -c "pipenv run tox"
-
-after_script:
-  - docker-compose -f docker-compose.yml -f docker-compose.travis.yml down
```
Comment (peakwinter) [inline .travis.yml]: Is the removal of this line necessary? I'd like to ensure that all resources for the job are truly cleaned up once the tests are executed...

Comment (maximeconnolly) [reply .travis.yml]: I removed it when I was debugging why the build errored out, its my mistake the line has been added back to the yaml file.

File: .travis.yml [hunk 2]
```diff
@@ -5,8 +5,10 @@ services:
 
 before_script:
   - docker-compose -f docker-compose.yml -f docker-compose.travis.yml up -d
+  - docker-compose ps
   - docker-compose exec web sh -c "apk add --no-cache git"
-  - docker-compose exec web sh -c "pipenv install -d tox"
```
Comment (peakwinter) [inline .travis.yml]: We should also keep this `-d` flag, that way pipenv doesn't go through the trouble unnecessarily of creating a local virtualenv when it's in the docker container.

**Review by peakwinter — CHANGES_REQUESTED** [review]

**Comment by peakwinter** [comment]
@maximest-pierre Just a quick merge conflict to resolve before I can merge! Thank you!

**Review by peakwinter — APPROVED** [review]

**Pull request merged**